src/main/java/it/trade/model/reponse/TradeItPreviewStockOrEtfOrderResponse.java

```diff
@@ -19,17 +15,32 @@
     @Expose
     public String orderId;
 
+    @SerializedName("warnings")
```

Comment:
From our documentation, it looks like it should be at the `orderDetails`level? (https://www.trade.it/documentation#PreviewTradeOrderDetails)

Reply:
Good point. I thought they were at the same level as the others.